### ISSUE TITLE
Support for .Net 6 Isolated with v4 functions

### ISSUE THREAD

**Issue by SyntaxWarrior** [yuka1984/azure-functions-extensions-swashbuckle#54]
Hi!
I'm wondering if there are any plans (and if so when) to support .net 6 Azure functions running in Isolated mode. 

P.S. I know this exists https://github.com/Azure/azure-functions-openapi-extension but that requires you to add attributes to everything creating a great risk of the swagger definition becoming out of sync with the code instead of generating the documents form the code like this project does.

**Comment by nwaoga** [comment]
Yes It also leaves function.cs code bloated with the main focus being attributes and still compiles if attributes are incorrect. Please can you make it so this can be generated by the xml  as in previous iterations
